doc/adrs/153_selection_dialog_tree_view.adoc

```diff
@@ -107,6 +118,8 @@ Root
   |_ (F)
 ```
 
+==== Frontend
```

Comment:
What about an update of the subscription that will modify the tree? You should also be able to keep the already expanded nodes expanded.

Reply:
IMO, keeping the expanded state locally to the user makes sense since we don't share the representation with others.